docs/how-to/specify-a-base.rst

```diff
@@ -0,0 +1,58 @@
+.. _how-to-specify-a-base:
+
+Specify a base
+==============
+
+All rocks are layered on top of a software base, which determines compatibility with
+their contents. When you craft a rock, you must specify its base, because it can't
+assume a default base.
+
+Bases are either an :ref:`Ubuntu LTS or interim release
+<explanation-bases-lts-and-interim-bases>`, or the :ref:`bare base
+<explanation-bases-bare-bases>`.
+
+
+Ubuntu LTS and interim bases
+----------------------------
+
+Specify an Ubuntu base with:
+
+.. code-block:: yaml
+    :caption: rockcraft.yaml
+
+    base: ubuntu@<version>
+
+For most rocks, select the latest Ubuntu LTS base, as your rock will benefit from that
```

Comment:
```suggestion
For most Ubuntu-based rocks, select the latest Ubuntu LTS base, as your rock will benefit from that
```

This is to emphasize that this sentence refers specifically to ubuntu rocks. "For most rocks" makes me think it includes bare-based too (even though this is a specific section)